GitHub.Collectors/PointCollectorInput.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.CloudMine.Core.Collectors.Error;
+using Microsoft.CloudMine.GitHub.Collectors.Model;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Microsoft.CloudMine.GitHub.Collectors
+{
+    [Serializable]
+    public class PointCollectorInput
+    {
+        public string Url { get; set; }
+        public virtual string RecordType { get; set; }
```

Comment:
Do we have any instantiations of PointCollectorInput? If not, why have virtual stuff?